pkg/fission-cli/cmd/client.go

```diff
@@ -142,3 +156,21 @@ func NewClient(opts ClientOptions) (*Client, error) {
 
 	return client, nil
 }
+
+// Fetch default namespace for inClusterConfig
+func getInClusterConfigamespace() string {
```

Comment:
Please fix function name. Also see if any reusable function for this already exists in client-go

Reply:
we don't have any reusable function.

Reply:
https://github.com/kubernetes/kubernetes/pull/63707